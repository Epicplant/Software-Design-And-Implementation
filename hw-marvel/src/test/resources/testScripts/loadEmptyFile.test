# Tests that a graph that has absolutely no nodes is loaded correctly

# Loads a new graph from a csv file
LoadGraph newGraph emptyFile.csv

# Lists all of the nodes of this new graph
ListNodes newGraph