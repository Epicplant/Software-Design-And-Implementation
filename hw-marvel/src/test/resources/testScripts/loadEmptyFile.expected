# Tests that a graph that has absolutely no nodes is loaded correctly

# Loads a new graph from a csv file
loaded graph newGraph

# Lists all of the nodes of this new graph
newGraph contains: